src/assets/stylesheets/containers/intro.sass

```
@@ -1,129 +1,158 @@
-.intro
-  margin: $m-xxl * 2
+.advert
 
-.slider
-  border:
-    top: $m-xs/2 solid rgba($color-white, 0.5)
-  display: flex
-  justify-content: space-between
+  &__btn
+    background-color: rgba($color-white, 0.4)
+    display: inline-block
+    font:
+      size: $font-size-base
+      weight: $font-weight-bold
+    margin-top: $m-xxl
+    padding: $m-md $m-xl
+    position: relative
+    text-align: center
+    text-decoration: none
+
+  &__details
+    border-left: 3px solid $color-white
+    font-size: $font-size-lg
+    margin-top: $m-xl
+    padding-left: $m-md
+
+    div:last-child
+      margin-top: $m-md
 
   &__header
-    border:
-      bottom: $m-xs/2 solid $color-white
-    text-transform: uppercase
-    padding-bottom: $m-xs
-    line-height: $line-height-xl
-    display: inline-block
+    font:
+      size: $font-size-xxl / 2
+      weight: $font-weight-bold
+    line-height: $line-height-sm
+
+  &__icon
+    background-color: $color-white
+    bottom: 0
+    color: $brand-primary
+    display: block
+    font-size: $font-size-lg
+    left: 100%
+    padding: $m-md
     position: absolute
-    top: 132px
+    top: 0
 
-  &__buttons
-    flex: 0 1 15%
-    display: flex
-    justify-content: center
-    position: relative
-    top: -3px
+  &__subheader
+    color: $color-white
+    font:
+      size: $font-size-xl
+      weight: $font-weight-bold
+    margin-top: $m-xl
+
+.main-container
+  margin:
+    left: $m-lg
+    right: $m-lg
+
+.primary-content
+  background:
+    color: $color-black
+    image: linear-gradient(to bottom, rgba(255,0,0,0), rgba(255,0,0,0), rgba(255,0,0,0), rgba(255,0,0,0), $color-gray-light), url('/b.jpg')
+  margin-bottom: $m-xl
+  padding-bottom: $m-xl
+
+.secondary-content
+  background-color: $color-gray-light
+  color: $color-black
+
+.slider
+  border-top: $m-xs/2 solid rgba($color-white, 0.5)
+  display: flex
+  justify-content: space-between
+  margin-bottom: $m-xxl * 2
+  position: relative
 
   &__button
-    flex: 0 1 33%
-    margin:
-      left: $m-xs
+    align-items: center
+    background-color: rgba($color-white, 0.4)
     border:
+      bottom: 0
+      left: 0
+      right: 0
       top: $m-xs/2 solid rgba($color-white, 0.5)
-    background-color: rgba($color-white, 0.4)
     color: rgba($color-white, 0.5)
-    font-weight: bold
-    margin:
-      left: $m-xs
     display: flex
-    justify-content: center
     flex-direction: column
-    align-items: center
+    flex: 0 1 33%
+    font-weight: $font-weight-bold
+    justify-content: center
+    margin-left: $m-xs
     max-height: 30%
 
-    &--active
-      @extend .slider__button
+    &.-active
       background-color: rgba($color-white, 0.7)
-      border:
-        top: $m-xs/2 solid $color-blue
+      border-top: $m-xs/2 solid $brand-selected
```

You are only changing the color, so 
```suggestion
      border-top-color: $brand-selected
```